ui-v2/ember-cli-build.js

```diff
@@ -65,6 +65,8 @@ module.exports = function(defaults) {
   // modules that you would like to import into your application
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
+  app.import('node_modules/text-encoding/lib/encoding-indexes.js', {outputFile: 'assets/encoding-indexes.js'})
```

Comment:
I don't think I've run across `outputFile` in ember-cli docs - that's really handy. 👌

Reply:
I vaguely remember reading about them in a GH issue, that was an issue saying they were missing from the documentation 😄

Reply:
Extra note I figured on this yesterday, might be helpful if you ever need to use this. If you do this multiple times to the same `outputFile` it concats the sources together into the same file:

```
app.import('source-1', {outputFile: 'all.js'})
app.import('source-2', {outputFile: 'all.js'})
```

At some point 🔜 I'll probably merge the 2 files above into the same file to save a request.